app/javascript/components/molecules/FormGridRow.js

```diff
@@ -11,4 +11,24 @@ const FormGridRow = ({ label, id, name, placeholder, describeId, note, error })
   )
 }
 
+const FormGrid = {}
```

Comment:
@fddayan Why is this block of code here as well as in the file above?

Reply:
forgot to remove it :P